cmd/goal/messages.go

```diff
@@ -56,38 +56,38 @@ const (
+	infoNodeStart                      = "Algorand node successfully started!"
+	infoNodeAlreadyStarted             = "Algorand node was already started!"
+	infoNodeDidNotRestart              = "Algorand node did not restart. The node is still running!"
+	infoTryingToStopNode               = "Trying to stop the node..."
+	infoNodeShuttingDown               = "Algorand node is shutting down..."
+	infoNodeSuccessfullyStopped        = "The node was successfully stopped."
+	infoNodeStatus                     = "Last committed block: %d\nTime since last block: %s\nSync Time: %s\nLast consensus protocol: %s\nNext consensus protocol: %s\nRound for next consensus protocol: %d\nNext consensus protocol supported: %v"
+	catchupStoppedOnUnsupported        = "Last supported block (%d) is committed. The next block consensus protocol is not supported. Catchup service is stopped."
+	infoNodeCatchpointCatchupStatus    = "Last committed block: %d\nSync Time: %s\nCatchpoint: %s"
+	infoNodeCatchpointCatchupAccounts  = "Catchpoint total accounts: %d\nCatchpoint accounts processed: %d\nCatchpoint accounts verified: %d"
+	infoNodeCatchpointCatchupBlocks    = "Catchpoint total blocks: %d\nCatchpoint downloaded blocks: %d"
+	nodeLastCatchpoint                 = "Last Catchpoint: %s"
+	errorNodeCreationIPFailure         = "Parsing passed IP %v failed: need a valid IPv4 or IPv6 address with a specified port number"
+	errorNodeNotDetected               = "Algorand node does not appear to be running: %s"
+	errorNodeStatus                    = "Cannot contact Algorand node: %s"
+	errorNodeFailedToStart             = "Algorand node failed to start: %s"
+	errorNodeRunning                   = "Node must be stopped before writing APIToken"
+	errorNodeFailGenToken              = "Cannot generate API token: %s"
+	errorNodeCreation                  = "Error during node creation: %v"
+	errorNodeManagedBySystemd          = "This node is using systemd and should be managed with systemctl. For additional information refer to https://developer.algorand.org/docs/run-a-node/setup/install/#installing-algod-as-a-systemd-service"
+	errorKill                          = "Cannot kill node: %s"
+	errorCloningNode                   = "Error cloning the node: %s"
+	infoNodeCloned                     = "Node cloned successfully to: %s"
+	infoNodeWroteToken                 = "Successfully wrote new API token: %s"
+	infoNodePendingTxnsDescription     = "Pending Transactions (Truncated max=%d, Total in pool=%d): "
+	infoNodeNoPendingTxnsDescription   = "None"
+	infoDataDir                        = "[Data Directory: %s]"
+	errLoadingConfig                   = "Error loading Config file from '%s': %v"
+	errorNodeFailedToShutdown          = "Unable to shut down node: %v"
+	errorCatchpointLabelParsingFailed  = "The provided catchpoint is not a valid one"
+	errorUnableToLookupCatchpointLabel = "Unable to fetch catchpoint lable"
```

Comment:
I'd personally add ": %w" at the end and add the original error into the message.
Without this the error checking in `TestGetMissingCatchpointLabel` does not make sense since it is not propagated to the end user.

Reply:
The error text also has a spelling error: "label" not "lable"